src/main/java/morgana/commands/AddCommand.java

```diff
@@ -0,0 +1,46 @@
+package morgana.commands;
+
+import morgana.exceptions.MorganaException;
+import morgana.storage.Storage;
+import morgana.task.Task;
+import morgana.task.TaskList;
+import morgana.ui.Ui;
+
+/**
+ * Represents an abstract command to add a task to the task list.
+ * Concrete subclasses should implement the {@link #createTask(String)} method
+ * to define how the task is created based on the user's input.
+ */
+public abstract class AddCommand extends Command {
+    private final String args;
```

Comment:
Should this be in caps instead since it is a final variable?

Reply:
`args` is not a constant so it does not need to be capitalized.